source/assets/js/components/fef-generic-expander.js

```diff
@@ -0,0 +1,138 @@
+import {KEYCODES} from '../utils/fef-keycodes';
+
+const HOOK_CLASS        = '.js-genex';
+const TOGGLE_CLASS      = '.js-genex-toggle';
+const ANIMATION_DEFAULT_EASING = 'easeInOutCubic';
+
+let ANIMATION_DEFAULT_DURATION = 400;
+
+if (window.matchMedia('(prefers-reduced-motion)').matches) {
+    ANIMATION_DEFAULT_DURATION = 0;
+}
+
+export function init() {
+    $(HOOK_CLASS).each((index, element) => {
+        new FefGenericExpander(element);
+    });
+}
+export class FefGenericExpander {
+    constructor(element) {
+        this.$element = $(element);
+        this.openToggleClass = this.$element.attr('data-genex-open-toggle-class') || 'js-genex-toggle-open';
+        this.openPanelClass  = this.$element.attr('data-genex-open-panel-class')  || 'js-genex-panel-open';
+
+        this.eventSource = this.$element.attr('data-event-source');
+        this.eventValue = this.$element.attr('data-event-value');
+
+        this.initA11Y();
+        this.bindEvents();
+
+        this.isTogglingAllowed = true;
+    }
+
+    initA11Y() {
+        // set aria-expand attribute
```

Comment:
Zeil 35 und 38-40 loopen doch über das gleiche Set, oder? Ist das bewusst aufgeteilt worden?

Und: Es werden alle Elemente mit dieser Klasse genommen, nicht nur die in diesem Generic-Expander! (--> mit `$(..., this.$element)` einen Kontext geben)